### ISSUE TITLE
schematron heeft geen constraints als de waarde uit codelijst MediaType komt

### ISSUE THREAD

**Issue by Harm-Olthof** [Geonovum/etf-test-projects-nl#34]
Volgens https://docs.geostandaarden.nl/md/mdprofiel-iso19115/#protocol is het verplicht een protocol op te nemen als er een URL gegeven wordt. Het stelt verder: "Indien de URL een 'accessPoint' betreft, dient een waarde geselecteerd te worden uit de codelijst Protocol", waarbij de waarden verderop in het document worden gegeven, bijvoorbeeld: png  
Voor zover ik  etf-test-projects-nl/src/metadata/Nederlands profiel op ISO 19115 v13 INSPIRE 2014/schematron.sch begrijp wordt daar echter alleen op de constraint voor codelijst Protocol gecontroleerd. Dit bekent dat als je bijvoorbeeld opneemt `<gco:CharacterString>png</gco:CharacterString>` of  `<gmx:Anchor xlink:href="https://www.iana.org/assignments/media-types/image/png">png</gmx:Anchor>` dit in de online validator een fout oplevert.

**Comment by thijsbrentjens** [comment]
Dank voor de melding. Voor het profiel versie 2.0 (https://docs.geostandaarden.nl/md/mdprofiel-iso19115/) hebben we nog geen schematron regels. Daar zijn we nu mee bezig. De schematron regels waar je naar verwijst, zijn van versie 1.3 van het metadata profiel.
We zullen dit punt opnemen.

**Comment by Harm-Olthof** [comment]
@thijsbrentjens dank voor de toelichting. Ik probeer metadata te valideren met deze validator: http://validatie.geostandaarden.nl/metadata/dataset (Link komt van de Geonovum site). Dat is toch de vigerende validator? Desalniettemin lukt het niet de metadata gevalideerd te krijgen als ik een mediatype opneem uit de lijst https://docs.geostandaarden.nl/md/mdprofiel-iso19115/#codelist-mediatypes
Vandaar dat ik op zoek ging naar hoe dit wordt afgedwongen en hier terecht kwam. De vraag blijft dan: worden deze schematron regels gebruikt in de door mij aangehaalde validator of moet ik een andere validator gebruiken?

**Comment by thijsbrentjens** [comment]
Klopt, dat is de vigerende validator. Wat ik bedoelde te zeggen met mijn eerdere opmerking: we hebben nog geen validator voor het nieuwe profiel.
